internal/util/cluster_mapping_test.go

```diff
@@ -84,69 +84,61 @@ func TestGetClusterMappingInfo(t *testing.T) {
-			clusterID:               "site-a-clusterid",
-			createMappingConfigFile: false,
-			mappingFilecontent:      []byte{},
-			expectedData:            nil,
-			expectErr:               false,
+			name:               "mapping file not found",
+			clusterID:          "site-a-clusterid",
+			mappingFilecontent: []byte{},
+			expectedData:       nil,
+			expectErr:          false,
 		},
 		{
-			name:                    "mapping file found with empty data",
-			clusterID:               "site-a-clusterid",
-			createMappingConfigFile: true,
-			mappingFilecontent:      []byte{},
-			expectedData:            nil,
-			expectErr:               false,
+			name:               "mapping file found with empty data",
+			clusterID:          "site-a-clusterid",
+			mappingFilecontent: []byte{},
+			expectedData:       nil,
+			expectErr:          false,
 		},
 		{
-			name:                    "cluster-id mapping not found",
-			clusterID:               "site-a-clusterid",
-			createMappingConfigFile: true,
-			mappingFilecontent:      mappingFileContent,
-			expectedData:            nil,
-			expectErr:               false,
+			name:               "cluster-id mapping not found",
+			clusterID:          "site-a-clusterid",
+			mappingFilecontent: mappingFileContent,
+			expectedData:       nil,
+			expectErr:          false,
 		},
 		{
-			name:                    "site2-storage cluster-id mapping",
-			clusterID:               clusterIDOfSite2,
-			createMappingConfigFile: true,
-			mappingFilecontent:      mappingFileContent,
-			expectedData:            &expectedSite2Data,
-			expectErr:               false,
+			name:               "site2-storage cluster-id mapping",
+			clusterID:          clusterIDOfSite2,
+			mappingFilecontent: mappingFileContent,
+			expectedData:       &expectedSite2Data,
+			expectErr:          false,
 		},
 		{
-			name:                    "site1-storage cluster-id mapping",
-			clusterID:               clusterIDOfSite1,
-			createMappingConfigFile: true,
-			mappingFilecontent:      mappingFileContent,
-			expectedData:            &expectedSite1To2Data,
-			expectErr:               false,
+			name:               "site1-storage cluster-id mapping",
+			clusterID:          clusterIDOfSite1,
+			mappingFilecontent: mappingFileContent,
+			expectedData:       &expectedSite1To2Data,
+			expectErr:          false,
 		},
 		{
-			name:                    "site3-storage cluster-id mapping",
-			clusterID:               clusterIDOfSite3,
-			createMappingConfigFile: true,
-			mappingFilecontent:      mappingFileContent,
-			expectedData:            &expectedSite3To2Data,
-			expectErr:               false,
+			name:               "site3-storage cluster-id mapping",
+			clusterID:          clusterIDOfSite3,
+			mappingFilecontent: mappingFileContent,
+			expectedData:       &expectedSite3To2Data,
+			expectErr:          false,
 		},
 	}
-	for _, tt := range tests {
+	for i, tt := range tests {
+		i := i
```

Comment:
please use different variable names, like `currentI` and `currentTT` so that it is less confusing